### DISCUSSION TITLE
Java swing is still available!

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#271] [Show and tell]
It can be ugly, but, it might be helpful...
![image](https://user-images.githubusercontent.com/3694604/227751784-24947fe5-995b-4331-b5b1-c848d7ec9afe.png)

**Comment by villares** [comment]
more examples here: https://github.com/villares/processing-python/tree/main/swing_examples

**Reply by hx2A** [reply]
These aren't ugly, they are beautiful! I tried a few of them, they are quite nice. I imagine the `JOptionPane` stuff might be a bit tedious for some. Maybe these can be packaged up into some easy to access functions?

**Reply by ismailcanvarli** [reply]
thx
